DDDEastAnglia/Views/Shared/_Layout.cshtml

```diff
@@ -1,107 +1,108 @@
+                </a>
+                <div class="btn-group pull-right">
+                    @Html.Partial("_LoginPartial")
                 </div>
+                @Html.Action("RenderMenu", "NavigationBar")
             </div>
         </div>
+    </div>
+    <div class="container-fluid">
         <div class="container-fluid">
-            <div class="container-fluid">
-                <div class="row-fluid">
-                    <div class="span8">
-                        @RenderSection("featured", required: false)
-                        <div class="row-fluid">
-                            @RenderBody()
-                        </div>
-                    </div><!--/span9-->
-                    <div class="span4" id="sidebar">
-                        <div class="well">
-                            <section class="row-fluid">
-                                <h1>When and Where?</h1>
-                                <p>
-                                    @Html.Action("ConferenceDate", "Timeline")
-                                    <br />
-                                    @Html.Action("ConferenceTime", "Timeline")
-                                </p>
-                                <p>
-                                    <a href="http://westroad.org">West Road Concert Hall</a><br />
-                                    11 West Road<br />
-                                    Cambridge<br />
-                                    CB3 9DP
-                                </p>
-                            </section>
-
-                            @Html.Action("Details", "Timeline")
+            <div class="row-fluid">
+                <div class="span8">
+                    @RenderSection("featured", required: false)
+                    <div class="row-fluid">
+                        @RenderBody()
+                    </div>
+                </div><!--/span9-->
+                <div class="span4" id="sidebar">
+                    <div class="well">
+                        <section class="row-fluid">
+                            <h1>When and Where?</h1>
+                            <p>
+                                @Html.Action("ConferenceDate", "Timeline")
+                                <br />
+                                @Html.Action("ConferenceTime", "Timeline")
+                            </p>
+                            <p>
+                                <a href="http://westroad.org">West Road Concert Hall</a><br />
+                                11 West Road<br />
+                                Cambridge<br />
+                                CB3 9DP
+                            </p>
+                        </section>
 
-                            @Html.Partial("_Sponsors")
-                        </div>
+                        @Html.Action("Details", "Timeline")
+                        @{
+                            Html.RenderAction("Sidebar", "Sponsor");
```

Comment:
Ok.. I remember now.. the line above is rendering a link.. where as the RenderAction is actually because we're calling the Sidebar action in the Sponsor Controller.. which requires access to the SponsorQuery.. Hope that makes sense with the Html.RenderAction.

see http://www.dotnet-tricks.com/Tutorial/mvc/Q8V2130113-RenderPartial-vs-RenderAction-vs-Partial-vs-Action-in-MVC-Razor.html